server/resp/src/main/java/org/infinispan/server/resp/Consumers.java

```diff
@@ -20,12 +23,18 @@ private Consumers() {
 
    }
 
-   static final byte[] OK = "+OK\r\n".getBytes(StandardCharsets.US_ASCII);
-   static final byte[] MATCHES = "matches".getBytes(StandardCharsets.US_ASCII);
-
    public static final BiConsumer<Object, ByteBufPool> OK_BICONSUMER = (ignore, alloc) -> alloc.acquire(OK.length)
          .writeBytes(OK);
 
+   public static final BiConsumer<Object, ByteBufPool> SYNTAX_ERROR_BICONSUMER = (ignore, alloc) -> alloc.acquire(SYNTAX_ERROR.length)
+         .writeBytes(SYNTAX_ERROR);
+
+   public static final BiConsumer<String, ByteBufPool> WRONG_ARGUMENT_COUNT_BICONSUMER = (command, alloc) -> alloc.acquire(46 + command.length())
+         .writeBytes(String.format("ERR wrong number of arguments for '%s' command", command).getBytes(StandardCharsets.US_ASCII));
+
+   public static final BiConsumer<String, ByteBufPool> VALUE_NOT_INTEGER_BICONSUMER = (command, alloc) -> alloc.acquire(VALUE_NOT_INTEGER.length)
+         .writeBytes(VALUE_NOT_INTEGER);
+
```

Comment:
There is a `RespErrorUtil` that holds the error messages and calls the byte buffer to execute the allocation logic. Could add the same logic there and reference from here, to keep the `BiConsumer`.